bcda/cli_test.go

```diff
@@ -106,6 +106,86 @@ func (s *CLITestSuite) TestCreateACO() {
 	buf.Reset()
 }
 
+func (s *CLITestSuite) TestImportCCLF0() {
+	assert := assert.New(s.T())
+
+	cclf0filePath := "../shared_files/cclf/T.A0001.ACO.ZC0Y18.D181120.T1000011"
+	cclf0metadata, err := getCCLFFileMetadata(cclf0filePath)
+	cclf0metadata.filePath = cclf0filePath
+	assert.Nil(err)
+
+	cclf8filePath := "../shared_files/cclf/T.A0001.ACO.ZC8Y18.D181120.T1000009"
+	cclf8metadata, err := getCCLFFileMetadata(cclf8filePath)
+	cclf8metadata.filePath = cclf8filePath
+	assert.Nil(err)
+
+	cclf9filePath := "../shared_files/cclf/T.A0001.ACO.ZC9Y18.D181120.T1000010"
+	cclf9metadata, err := getCCLFFileMetadata(cclf9filePath)
+	cclf9metadata.filePath = cclf9filePath
+	assert.Nil(err)
+
+	// positive
+	_, err = importCCLF0(cclf0metadata)
+	assert.Nil(err)
+	err = importCCLF8(cclf8metadata)
+	assert.Nil(err)
+	err = importCCLF9(cclf9metadata)
+	assert.Nil(err)
+
+	// negative
+	cclf0metadata = cclfFileMetadata{}
+	_, err = importCCLF0(cclf0metadata)
+	assert.NotNil(err)
+
+	// missing cclf8 and or 9 from cclf0
+	cclf0filePath = "../shared_files/cclf0_MissingData/T.A0001.ACO.ZC0Y18.D181120.T1000011"
+	cclf0metadata, err = getCCLFFileMetadata(cclf0filePath)
+	cclf0metadata.filePath = cclf0filePath
+	assert.Nil(err)
+	_, err = importCCLF0(cclf0metadata)
+	assert.NotNil(err)
+}
+
+func (s *CLITestSuite) TestValidate() {
+	assert := assert.New(s.T())
+
+	cclf0filePath := "../shared_files/cclf/T.A0001.ACO.ZC0Y18.D181120.T1000011"
+	cclf0metadata, err := getCCLFFileMetadata(cclf0filePath)
+	cclf0metadata.filePath = cclf0filePath
+	assert.Nil(err)
+
+	cclf8filePath := "../shared_files/cclf/T.A0001.ACO.ZC8Y18.D181120.T1000009"
+	cclf8metadata, err := getCCLFFileMetadata(cclf8filePath)
+	cclf8metadata.filePath = cclf8filePath
+	assert.Nil(err)
+
+	cclf9filePath := "../shared_files/cclf/T.A0001.ACO.ZC9Y18.D181120.T1000010"
+	cclf9metadata, err := getCCLFFileMetadata(cclf9filePath)
+	cclf9metadata.filePath = cclf9filePath
+	assert.Nil(err)
+
+	// positive
+	cclfvalidator, err := importCCLF0(cclf0metadata)
```

Comment:
Same as comment above here-- it will test `validate()` more specifically if we create the exact `cclfFileMetadata` structs and validator maps that we want the test to use.